.github/workflows/diff_report.yml

```diff
@@ -104,6 +104,17 @@ jobs:
         ref: master
         path: checkstyle
         fetch-depth: 0
+
+     # set main checkstyle repo as an upstream
+     # Diff report will be generated taking upstream's master branch as a base branch
+     - name: set upstream
+       run: |
+         cd checkstyle
+         bash
+         MAIN_REPO_GIT_URL="https://github.com/checkstyle/checkstyle.git"
+         git remote add original_checkstyle $MAIN_REPO_GIT_URL\
```

Comment:
`original_checkstyle` ==> `upstream-master`

Reply:
@romani 
> "Base branch: upstream/master", may be it is better to name remote as original_checkstyle to make it like "Base branch: original_checkstyle/master". What do you think?


see: https://github.com/checkstyle/checkstyle/pull/8828#issuecomment-691673694
@strkkk suggested to change `upstream` to `original_checkstyle` so that it looks much relatable to diff report home page. I too liked it.

Should I change it again? 
It will look like `upstream-master/master` after this change.

Reply:
current-report homepage: https://abhishek-kumar09.github.io/github-actions-report-fetch/

Reply:
@strkkk, original and origin is kind of the same. It should be better to make it clear that we do not use origin of user at all.
Please share your thoughts on this

Reply:
@romani as for me, original_checkstyle is clearer than "origin" or "upstream"
I do not insist on this naming.

Reply:
what should I do here?

Reply:
@Abhishek-kumar09 please change to what @romani suggested to avoid long discussions and not blocking you

Reply:
Done